---
geometry_msgs/PoseStamped object_pose
bool success